hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -504,6 +511,20 @@ public void close() throws Exception {
     }
   }
 
+  /**
+   * Return the timestamp of the latest synced instant.
+   */
+  @Override
+  public Option<String> getLatestSyncedInstantTime() {
```

Comment:
could n't we call `metadata.getSyncedInstantTime()` here?

Reply:
Yep. Remove this method.